src/builtins/core/date.rs

```diff
@@ -600,8 +600,52 @@ impl PlainDate {
             .with_calendar(self.calendar.identifier(), display_calendar)
             .build()
     }
+
+    #[inline]
+    pub fn to_zoned_date_time(
+        &self, 
+        tz: TimeZone,
+        pt: Option<PlainTime>, 
```

Comment:
nit: `plain_time` over `pt`. `tz` is debatable but also currently used elsewhere, so that can be left as is.